src/frontend/magnify/src/components/rooms/RegisterRoom/RegisterRoom.tsx

```diff
@@ -0,0 +1,68 @@
+import { defineMessages, useIntl } from 'react-intl';
+import React, { useState } from 'react';
+import { Box, Button, Card, Layer, Text } from 'grommet';
+import { Add } from 'grommet-icons';
+import RegisterRoomForm from '../RegisterRoomForm';
+
+const messages = defineMessages({
+  registerNewRoomLabel: {
+    id: 'components.rooms.registerRoom.registerNewRoomLabel',
+    defaultMessage: 'Register new room',
+    description: 'Label for the button to register a new room',
+  },
+});
+
+const RegisterRoom = () => {
+  const intl = useIntl();
```

Comment:
sometomes you use `uneIntl`, sometimes `useTranslation`. The abstraction you made is IMO a pitfall